ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -40,7 +40,7 @@ sealed abstract class Queries {
     sql"""
       CREATE TABLE
         contract
-        (contract_id """ ++ contractIdType ++ sql""" NOT NULL PRIMARY KEY
+        (contract_id """ ++ contractIdType ++ sql""" NOT NULL CONSTRAINT contract_k PRIMARY KEY
```

Comment:
Not at all opposed to this quite the opposite but is this needed for something else in this PR or just a drive-by change?

Reply:
Drive-by, it seemed nicer to have error messages tell you _which_ constraint failed.